libs/map/framework.cpp

```diff
@@ -3369,7 +3369,7 @@ uint32_t Framework::GetTimeoutForReason(ProductsPopupCloseReason reason) const
   return kPopupCloseTimeout;
 }
 
-std::string_view Framework::ToString(ProductsPopupCloseReason reason) const
+std::string_view Framework::ToString(ProductsPopupCloseReason reason)
```

Comment:
Why is it in the framework? Can be in a separate file or a product-related file.